### PR TITLE
Add image alt attribute to the from the top of the post contents

### PR DESCRIPTION
**Pull request by bms8197**
## Description

e.g. Fixes #(issue)

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## How has this been tested

- [x] I have run `bash ./tools/test.sh --build` (at the root of the project) locally and passed
- [x] I have tested this feature in the browser

### Test Configuration
- Browser type & version: Safari v14.0.1
- Operating system: MacOS Catalina v10.15.7
- Bundler version: 2.2.7
- Ruby version: 2.5.5p15
- Jekyll version: 4.2.0

### Checklist

- [x] My code follows the [Google style guidelines](https://google.github.io/styleguide/)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by bms8197** [comment]
I think I've already deleted the old PR...

**Comment by cotes2020** [comment]
> I think I've already deleted the old PR...

GitHub does not allow anyone to delete a PR, event the owner of the repos  :)

File: _posts/2019-08-08-write-a-new-post.md [hunk 1]
```diff
@@ -88,10 +88,20 @@ If you want to add an image to the top of the post contents, specify the url for
...
-image: /path/to/image-file
+image:
+  src: /path/to/image-file
 ---
 ```
 
+### Image alt attribute
+
+If you want to add the "alt attribute" to the image at the top of the post contents, specify it by:
+```yaml
```
Comment (cotes2020) [inline _posts/2019-08-08-write-a-new-post.md:99]: Almost good enough to be merged, but a blank line should be added before the code block ```` ```yaml ````

File: _posts/2019-08-08-write-a-new-post.md [hunk 2]
```diff
@@ -88,10 +88,20 @@ If you want to add an image to the top of the post contents, specify the url for
...
+---
+image:
+  alt: image alternative text
+```
```
Comment (cotes2020) [inline _posts/2019-08-08-write-a-new-post.md:103]: Change the ```` ``` ```` to `---`

**Review by cotes2020 — CHANGES_REQUESTED** [review]

**Pull request merged**